### PR TITLE
List the handlers on startup

### PR DESCRIPTION
**Pull request by jithinjk**
Fairly new to this community. Not sure if this counts as a feature.
capture

- List route handlers at startup,
- Also, number of handlers running.

Would be nice to have this to get better understanding of handlers running.

Sorry for multiple commit builds, new to travis-ci.

### CODE REVIEW

**Comment by thinkerou** [comment]
Maybe you should squash commits.

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ git:
   depth: 3
 
 install:
+  - go get golang.org/x/crypto/ssh/terminal
```
Comment (thinkerou) [inline .travis.yml]: use `govendor` in `Makefile`?

Comment (jithinjk) [reply .travis.yml]: will fix these issues

**Comment by appleboy** [comment]
@jithinjk Can you post the screenshots?

**Comment by jithinjk** [comment]
Attached screenshot. [Capture](https://user-images.githubusercontent.com/1090114/31733999-ead9a278-b45a-11e7-9f44-9ea4b4320cd3.PNG)

**Comment by thinkerou** [comment]
#1729 has added `HandlerNames()` method, the pr have any worth? thanks! I think we should closing the pr, thanks!

**Comment by appleboy** [comment]
conflicts.

**Comment by thinkerou** [comment]
@appleboy we still need the pull request, I think we should not use it, because it will depend on one package, the pull request have not clear useful, and we have had `HandlerNames` method. thanks!

**Comment by appleboy** [comment]
this should be pending now?

**Comment by thinkerou** [comment]
closing

**Pull request closed without merging**